docs/maker.md

```diff
@@ -0,0 +1,141 @@
+# Maker CLI Documentation
+
+The maker CLI is a command-line interface for interacting with the maker server in the Coinswap protocol. It allows you to manage your maker wallet, view balances, and perform various operations.
```

Comment:
Explain what a maker server is.
- its a server that swaps coins with other client, earning a fee.
- The server uses `makerd/ckaer-lci` architecture. `makerd` is the daemon. `maker-cli` is a command line tool to operate the damon. its very similar to `bitcoind/bitcoin-cli`.
- `makerd` should be ideally run in a 24*7 online systems. Makerd privatekeys are online in hot wallet, so server side security has to be maintained by the users.

Then structure the doc into.

- Installation.
- Setting up Bitcoind (this will have a blank link)
- Running and connecting `makerd` to `bitcoind` rpc.
- Getting initial balance to create the fidelity bond.
- using `maker-cli` and get various details. Show the command outputs.